engine/src/main/java/org/terasology/logic/players/MenuControlSystem.java

```diff
@@ -27,6 +27,7 @@
 import org.terasology.input.Keyboard;
```

Comment:
This import and another one (`KeyDownEvent`) are now unused and can be taken out (there's also some empty javadoc and an older year if you feel like fixing!)